bigquery_etl/backfill/shredder_mitigation.py

```diff
@@ -29,11 +52,11 @@ class ColumnStatus(Enum):
 
 
 class DataTypeGroup(Enum):
-    """Data types in BigQuery. Not including ARRAY and STRUCT as these are not expected in aggregate tables."""
+    """Data types in BigQuery. Not supported: TIMESTAMP, ARRAY and STRUCT, which are dtypes currently not expected in aggregates."""
 
     STRING = ("STRING", "BYTES")
     BOOLEAN = "BOOLEAN"
-    NUMERIC = (
+    INTEGER = (
```

Comment:
Numeric is not an integer though? It's closer to a float?

Reply:
Indeed numeric covers most of the types, including integer. I'd propose:
 INTEGER / BIGINT
 NUMERIC
 FLOAT
and let the other types be added incrementally as aggreggates require them, since most of these types are in client-level tables and not in aggregates. What do you think?

Reply:
not sure I understand your suggestion?

Do you mean
NUMERIC = (integer, numeric, float)
or 
INTEGER = (integer, int64, bigint,..)
NUMERIC = (numeric, bignumeric,...)
FLOAT = (float, double, ..)
?
fine either way

Reply:
Not quite. I mean
INTEGER = INTEGER, INT64, INT
NUMERIC = NUMERIC
FLOAT = FLOAT